src/components/graph/graph.ts

```diff
@@ -111,17 +111,22 @@ export default class GraphComponent {
     this._c10 = d3.scaleOrdinal(d3.schemeCategory10)
 
     this._actionStream = actionStream
+    let isSingleClick: boolean = true
 
     this._svg = d3.select(host)
       .append('svg')
       .attr('width', dimensions.width)
       .attr('height', dimensions.height)
       .on('click', () => {
         d3.event.stopPropagation()
-        this._actionStream!.next(new BackgroundClickAction())
+        isSingleClick = true
+        setTimeout(() => { if (isSingleClick) {
```

Comment:
format this inline function nicely and drop the extra parens around the `if` block

Reply:
not sure what you mean by this, if you're referring to the curly braces right after the '=>' then they are necessary